clients/src/main/java/org/apache/kafka/common/requests/DescribeConfigsResponse.java

```diff
@@ -248,6 +321,14 @@ public DescribeConfigsResponse(Struct struct) {
                 String configName = configEntriesStruct.getString(CONFIG_NAME_KEY_NAME);
                 String configValue = configEntriesStruct.getString(CONFIG_VALUE_KEY_NAME);
                 boolean isSensitive = configEntriesStruct.getBoolean(IS_SENSITIVE_KEY_NAME);
+                ConfigType type = null;
```

Comment:
`ConfigType.UNKNOWN` instead of null since we use that as the default elsewhere?